### DISCUSSION TITLE
Memory leak in `diffusive_size_factor_dns`

### DISCUSSION THREAD

**Discussion by oscarhx** [PMEAL/porespy#834] [Q&A]
Hello Profs.
As stated in the title,  the 64G memory goes out when the number of pore pairs reaching several thousands, far behind the number of pore pairs and conductivity required for AI trainning.

**Comment by jgostick** [comment]
This should be fixed btw.  There was a bug in openpnm that made a copy of the Project when a slice was used access the objects (as opposed to just a regular index).  We've pushed that fix to OpenPNM, so now if you run this with the latest version of OpenPNM install it should work.
